SpatialGDK/Documentation/toc.md

```diff
@@ -1,29 +1,35 @@
 - <h3>SpatialOS GDK for Unreal</h3>
```

Comment:
Not looking at this as there's some changes going on as we are losing the tutorial edit.